### PR TITLE
fix(condo): DOMA-9421 fix error throwing and processing if some cells…

### PR DESCRIPTION
**Pull request by AleX83Xpert**
… are nil

### CODE REVIEW

File: apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js [hunk 1]
```diff
@@ -39,7 +44,7 @@ class DomaMetersImporter extends AbstractMetersImporter {
         return {
             address: row[0],
             addressInfo: {
-                unitType: this.mappers.unitType[row[2].toLowerCase()],
+                unitType: this.mappers.unitType[String(row[2]).toLowerCase()],
                 unitName: row[1],
             },
             accountNumber: row[3],
```
Comment (sitozzz) [inline apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js:50]: Is there or somewhere else in this class any mechanism of checking row length and corresponding data types? IMO looks quite dangerous

Comment (AleX83Xpert) [reply apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js:50]: This class receives `string[][]` data got from incoming file. But there was a erroneous file with cell below the main table. See description in the jira task.

Comment (AleX83Xpert) [reply apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js:50]: I do not think we should control string length here. It's better incapsulate it within models.

Comment (sitozzz) [reply apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js:50]: Ok, sounds reasonable

**Review by sitozzz — APPROVED** [review]

**Pull request merged**